etcdserver/etcdserverpb/rpc.proto

```diff
@@ -637,6 +638,8 @@ message AuthUserAddResponse {
 
 message AuthUserGetResponse {
   ResponseHeader header = 1;
+
+  repeated string roles = 2;
```

Comment:
hmm. for v2 auth, we used to just list the names of the roles. then users want the detailed information of the roles, now it becomes https://github.com/coreos/etcd/blob/master/Documentation/v2/auth_api.md#users-1. What do you think? Shall we include the detailed information of role here?